lib/graphql/schema/loader.rb

```diff
@@ -22,9 +22,13 @@ def load(introspection_result)
           types[type_object.name] = type_object
         end
 
-        query = types.fetch(schema.fetch("queryType").fetch("name"))
+        kargs = {}
+        [:query, :mutation, :subscription].each do |root|
+          type = schema["#{root}Type"]
+          kargs[root] = types.fetch(type.fetch("name")) if type
+        end
 
-        Schema.new(query: query, types: types.values)
+        Schema.define(**kargs)
```

Comment:
`types` was renamed to `orphan_types` inside `.define` because of the naming conflict with the `types` helper object (eg `types.String`). 

It's too bad that the new name doesn't imply that "it's ok to just put all the types here and we'll sort it out later". Maybe it could be improved!